man/systemd.network.xml

```diff
@@ -1490,6 +1490,17 @@ VRF=vrf1
 </programlisting>
     </example>
 
+    <example>
+      <title>MacVTap</title>
+      <para>MacVTap</para>
```

Comment:
can you extend the paragraph here a bit? Doesn't have to be too long, but should be at least one proper sentence with a verb